ngraph/CMakeLists.txt

```diff
@@ -199,7 +197,10 @@ if (WIN32)
     string(REPLACE "/W3" "/W0" CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
 endif()
 
-if (NOT WIN32 AND NGRAPH_WARNINGS_AS_ERRORS)
+if (NOT CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
+    if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU" AND (NOT CMAKE_CXX_COMPILER_VERSION VERSION_LESS 6.0))
```

Comment:
`(NOT CMAKE_CXX_COMPILER_VERSION VERSION_LESS 6.0)` -> `CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 6.0`

Reply:
I will fix it in the next PR.